File: ProyectoIntegrador_GaleanoAgustin/BackEnd/Portfolio/src/main/java/com/portfolio/Portfolio/Entity/Experiencia.java

package com.portfolio.Portfolio.Entity;

import javax.persistence.Entity;
import javax.persistence.GeneratedValue;
import javax.persistence.GenerationType;
import javax.persistence.Id;

@Entity
public class Experiencia {

    @Id
    @GeneratedValue(strategy = GenerationType.IDENTITY)
    private int id;
    private String nombreE;
    private String descripcionE;

    public Experiencia() {
    }

    public Experiencia(String nombre, String descripcionE) {
        this.nombreE = nombre;
        this.descripcionE = descripcionE;
    }

    public int getId() {
        return id;
    }

    public void setId(int id) {
        this.id = id;
    }

    public String getNombre() {
        return nombreE;
    }

    public void setNombre(String nombre) {
        this.nombreE = nombre;
    }

    public String getDescripcionE() {
        return descripcionE;
    }

    public void setDescripcionE(String descripcionE) {
        this.descripcionE = descripcionE;
    }

}

File: ProyectoIntegrador_GaleanoAgustin/FrontEnd/src/app/model/experiencia.ts
export class Experiencia {
    id?: number;
    nombreE: string;
    descripcionE: string;
    anioE: string;

    constructor(nombreE: string, descripcionE: string, anioE: string){

        this.descripcionE = descripcionE;
        this.nombreE    = nombreE;
        this.anioE = anioE;
    }
}
